prow/test/integration/config/prow/cluster/moonraker_service.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: moonraker
+  namespace: default
+spec:
+  selector:
+    app: moonraker
+  ports:
+  - name: http
+    port: 80
+    targetPort: 8080
+    protocol: TCP
+  - name: metrics
+    port: 9090
+    protocol: TCP
+  type: NodePort
```

Comment:
nit: I don't think this needs to be a `NodePort`. I'd expect the default of `ClusterIP` to be sufficient for all of our services. The only service I'd expect to need a `NodePort` is the service used by the KinD cluster's ingress controller.

Reply:
The reason is because the integration test (go program) runs *outside* of the KinD cluster and we don't want to bother with going through the NGINX ingress (we want the test code to be able to send requests into the moonraker service directly). This is also why the fakepubsub uses a nodeport as well (see https://github.com/kubernetes/test-infra/blob/61e54aba4190f13679683cc514efbd2c74e64e96/prow/test/integration/config/prow/cluster/fakepubsub.yaml#L44-L50).

Reply:
> we don't want to bother with going through the NGINX ingress

Then why are there updates to the ingress object to route traffic to moonraker?
We should either use the ingress or (if that is not suitable for some reason) go directly through the `NodePort`, but we have configuration for both.

Reply:
> Then why are there updates to the ingress object to route traffic to moonraker?

Hmm, good question. I think we already go through the ingress (via the ingress rule updates). Will update to use a ClusterIP instead of NodePort.

Reply:
I just noticed that `deck` also uses NodePort in our integration tests (it also has a corresponding ingress object). This is the example I followed for setting up `sub` and `gangway` also (these both use NodePort). I'll need to update these other ones in a follow-up.

Reply:
This PR uses ClusterIP now.